src/components/Good.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import classes from '../components/Good.module.css';
 
 export function Good(props) {
+const{className, ...rest} = props
```

Comment:
Missing formatting here. I would install prettier in your IDE to auto-format things like these. Super convenient!